apps/job-info-consumer/consumer/cmd/start.go

```diff
@@ -108,7 +109,11 @@ func start(cmd *cobra.Command, options *StartOptions) error {
 	}
 	log.Ctx(ctx).Debug().Msgf("libp2p connecting to: %s", peers)
 
-	libp2pHost, err := libp2p.NewHost(options.swarmPort)
+	prvKey, err := libp2p.GeneratePrivateKey(2048)
```

Comment:
Did the linter not complain?

Reply:
Oddly enough it didn't, nor did my precommit hook, which I am using before pushing. I assume this might be because the linter isn't configured to lint this package. Still, I missed this magic number and will make a constant for it in this package.